aptos-move/framework/aptos-framework/sources/configs/stake.move

```diff
@@ -1062,8 +1077,8 @@ module aptos_framework::stake {
         // Here we do multiplication before division to minimize rounding errors.
         let base_rewards =
             voting_power * validator_set_config.rewards_rate / validator_set_config.rewards_rate_denominator;
-        let rewards_amount = base_rewards * num_successful_votes / num_blocks;
-        if (rewards_amount > 0) {
+        if (base_rewards > 0 && num_successful_proposals > 0) {
```

Comment:
I think we can skip the base_rewards > 0 check now as it's guaranteed to not be 0 (rewards rate can't be 0 and voting power can't be 0 either)

Reply:
I would leave it just in case. I.e. if nothing else, in tests someone might configure voting_power or rewards_rate to be 0

Reply:
It's not possible because we explicitly validate rewards rate > 0. Voting power = 0 also shouldn't happen as well although I don't think we have explicit validation for that case (only that voting power > min stake but min stake can be 0)